pkg/storaged/content-views.jsx

```diff
@@ -454,9 +454,9 @@
         if (config) {
             type = C_("storage-id-desc", "Filesystem (encrypted)");
             used_for = mount_point;
-        } else if (block_stratis_locked_pool) {
+        } else if (block_stratis_stopped_pool) {
             type = _("Stratis member");
-            used_for = block_stratis_locked_pool;
+            used_for = block_stratis_stopped_pool;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18679-20230420-131050-f1115ac6-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)